client/src/pages/Home.tsx

```diff
@@ -1,19 +1,28 @@
+import { useNavigate } from "react-router-dom";
 import DisplayLogo from "../components/DisplayLogo";
+import { HOST_GAME_PATH, JOIN_GAME_PATH } from "./PagePaths.ts";
 
 const Home = () => {
+  const navigate = useNavigate();
   return (
     <div>
       <div>
         <DisplayLogo />
       </div>
       <div className="w-screen h-12 mt-8">
-        <button className="text-white bg-primary text-2xl font-bold w-1/3 rounded-xl h-full">
-          JOIN ROOM
+        <button
+          onClick={() => navigate(JOIN_GAME_PATH)}
```

Comment:
Can we use a [Link component](https://reactrouter.com/en/main/components/link), instead of a button with an onClick event. Keep the styling the same though.